pages/how_to_create_ptc.md

```diff
@@ -1,49 +1,112 @@
-- [ ] 0 	No unit testing.
-- [ ] 1 	Unit testing implemented for one example refactored class.
-- [ ] 2 	Two developers using unit testing process.
-- [ ] 3 	Unit testing developer documentation completed.
-- [ ] 4 	All developer trained to write unit tests.
-- [ ] 5 	New policy established: All refactored code is covered by unit tests.
+### User Story
+
+Generating a user story will help you keep you focused on your goal and
+provide context. 
+A user story is a statement that can be phrased as "As a ____, I want ____ so that ____."
+In this case. a good user story may be:
+
+*As a **software developer**, I want to **create a unit testing policy covering
+refactored code for my team** so that **I can verify that certain functions of
+my code are working as expected."***
+
+### A series of qualitative improvement statements
+
+Two guidelines for creating improvement statements are:
+
+1. Create steps that are agreed upon by your team
+2. Try to have an artifact or measurable outcome for each step (e.g., policy,
+   document)
+
+Each item in the list is a qualitative description that illustrates
+what it means to reach that score. Zero (0) should represent a starting state,
+and the highest number should represent fully achieving the overall goal,
+keeping in mind that each step may also incorporate outcomes or artifacts
+along the way to completion.
 
-## Step 4: Record Current PTC Values.
+### Final Card
 
-Now that you have your PTC, you can Use your PTC in meetings with your team to remind everyone of your goals, and to determine whether you have met them. As seen below, you can record baseline capabilities and track progress (0-5 are suggested) for each PTC.
+| Name         | Unit Testing|
+| :-------------: | :------------- |
+| Target         | Apply unit testing policy to newly refactored code |
+| :-------------: | :------------- |
+| User Story | *As a **software developer**, I want to **create a unit testing policy covering refactored code for my team** so that **I can verify that certain functions of my code are working as expected."*** |
+| :-------------: | :------------- |
+| Score         | Description |
+| :-------------: | :------------- |
+| 0 | No unit testing. |
+| 1 | Unit testing implemented for one example refactored class. |
+| 2 | Two developers using unit testing process. |
+| 3 | Unit testing developer documentation completed. |
+| 4 | All developer trained to write unit tests. |
+| 5 | New policy established: All refactored code is covered by unit tests. |
+
+## Step 4: Record Current PTC Values
+
+![Step 4 of PSIP](/practice-guides/assets/images/Step4Highlight.png)
+
+Now that you have your PTC, you can Use your PTC in meetings with your team to
+remind everyone of your goals, and to determine whether you have met them.
+As seen below, you can record baseline capabilities and track progress
+for each PTC.
 
 - [x] 0 	No unit testing.
 - [ ] 1 	Unit testing implemented for one example refactored class.
```

Comment:
Very minor thing: these checkboxes look a little cramped next to the numbers. Adding a space might make it cleaner.